### PR TITLE
Initial

### PR DESCRIPTION
**Pull request by KlausPopp**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,81 @@
...
+
+Find out the MAC address of the peer machine's NIC:
+```bash
+ip link show eth0
+```
+Record the value for `link/ether`, e.g. `dc:a6:32:51:0e:e7`
+
+Start server on peer machine with root priviledges. Specify the name of the peer machine's NIC:
+```bash
+python3 server/eth-test-raw-server.py eth0
```
Comment (FranziskaArndt) [inline README.md]: changed folder structure, call should be "python3 eth-test-raw-server.py eth0"

File: README.md [hunk 2]
```diff
@@ -1,2 +1,81 @@
...
+```
+
+### On the Client
+
+Start the client specifying the interface name of the NIC to test and the peer machine's MAC address.
+You must have root priviledges:
+```bash
+python3 /data/eth-test-raw/client/eth-test-raw-client.py enp5s0 dc:a6:32:51:0e:e7
```
Comment (FranziskaArndt) [inline README.md]: changed folder structure, call should be "python3 eth-test-raw-client.py enp5s0 dc:a6:32:51:0e:e7"

File: README.md [hunk 3]
```diff
@@ -1,2 +1,81 @@
...
+
+## Platform compatibility
+Tested only under Linux (Rasperian OS and Yocto arm64). May work on Windows.
+
+## How to Use
+
+Connect device with the NIC to be tested to a peer machine via Ethernet.
+You may have L2 switches in between, but no routers.
+
+Deploy this repository, or at least the folders `client`, `server` and common to both machines.
```
Comment (FranziskaArndt) [inline README.md]: Folder structure has changed

**Review by FranziskaArndt — CHANGES_REQUESTED** [review]
readme should be corrected

File: README.md (hunk 1 above)
Comment (KlausPopp) [reply README.md]: ok

File: README.md (hunk 2 above)
Comment (KlausPopp) [reply README.md]: ok

File: README.md (hunk 3 above)
Comment (KlausPopp) [reply README.md]: ok

**Review by FranziskaArndt — APPROVED** [review]

**Pull request merged**